### PR TITLE
Fix luminous filestore

### PR DESCRIPTION
**Pull request by guits**

### CODE REVIEW

**Comment by guits** [comment]
jenkins test this please

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-centos7-dedicated_journal

**Comment by guits** [comment]
jenkins test ceph_ansible-kraken-centos7-cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-kraken-centos7-cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-kraken-centos7-cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-kraken-centos7-cluster

**Comment by guits** [comment]
jenkins test jenkins test ceph_ansible-kraken-centos7-cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-kraken-centos7-cluster

**Comment by guits** [comment]
jenkins test this please

**Comment by guits** [comment]
jenkins test this please

**Comment by guits** [comment]
jenkins test this please

**Comment by guits** [comment]
jenkins test this please

**Comment by guits** [comment]
jenkins test this please

**Comment by guits** [comment]
jenkins test ceph_ansible-luminous-xenial-dmcrypt_journal_collocation

**Comment by guits** [comment]
@leseb `ceph_ansible-jewel-centos7-cluster` and `ceph_ansible-luminous-centos7-cluster` are both failing because of the known race condition. I reproduced the CI tests manually and they both passed successfully the tests.

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-centos7-dmcrypt_journal_collocation

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-centos7-cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-centos7-cluster

**Comment by guits** [comment]
jenkins test ceph_ansible-jewel-centos7-cluster

File: ceph-releases/jewel/ubuntu/14.04/daemon/osd_scenarios/osd_disk_prepare.sh [hunk 1]
```diff
@@ -76,5 +76,7 @@ function osd_disk_prepare {
   # watch the udev event queue, and exit if all current events are handled
   udevadm settle --timeout=600
 
+  sleep 5
```
Comment (leseb) [inline ceph-releases/jewel/ubuntu/14.04/daemon/osd_scenarios/osd_disk_prepare.sh]: why?

Comment (guits) [inline ceph-releases/jewel/ubuntu/14.04/daemon/osd_scenarios/osd_disk_prepare.sh]: got a better fix here https://github.com/ceph/ceph-docker/pull/729

**Pull request merged**